src/providers/authentication/authentication.ts

```diff
@@ -101,6 +101,13 @@ export class AuthenticationProvider {
   }
 
   public getAuthenticationToken = async (): Promise<string> => {
+
+    // TODO remove temporary solution which checks if token has expired manually then forces token expiry
```

Comment:
can you annotate with `// @TODO`

Reply:
@Cullima didnt know that was a thing, made change and pushing now.